vars/mavenTemplate.groovy

```diff
@@ -42,7 +42,7 @@ def call(Map parameters = [:], body) {
                                 ],
                                 resourceLimitMemory: '640Mi')],
                 volumes: [
-                        secretVolume(secretName: 'jenkins-maven-settings', mountPath: '/root/.m2'),
+                        secretVolume(secretName: 'jenkins-maven-settings', mountPath: '/home/jenkins/.m2'),
```

Comment:
same here can we rely on env.JENKINS_HOME or something similar?

Reply:
Same as above